release.yaml

```diff
@@ -1,8 +1,18 @@
+rancher-istio:
```

Comment:
These versions of rancher-istio and fleet should not be released on the future.
They are just for fixing the patching issues.